### PR TITLE
Alter table properties nemesis tests + adding ability to set number of monitors to 0 (docker)

### PR DESCRIPTION
**Pull request by bentsi**
Hello dear colleagues.

My first contribution: I've added some new alter table nemesis and added ability to set number of monitors to 0 when using docker backend. In addition I've added CassandraStressCmdParseError exception in order to easily understand when something is wrong with the cassandra-stress command.

Please review.
@larisau 
@amoskong 
CC: @roydahan

### CODE REVIEW

File: sdcm/nemesis.py [hunk 1]
```diff
@@ -445,18 +445,79 @@ def disrupt_nodetool_cleanup(self):
...
-              " gc_grace_seconds = {};".format(gc_grace_seconds)
-        self.target_node.remoter.run('cqlsh -e "{}" {}'.format(cmd, self.target_node.private_ip_address), verbose=True)
+        self._modify_table_property(name="gc_grace_seconds", val=random.randint(216000, 864000))
+
+    def disrupt_modify_table_caching(self):
+        prop_val = dict(
+            keys=random.choice(["NONE", "ALL"]),
+            rows_per_partition=random.choice(["NONE", "ALL", random.randint(1, 10000)])
+        )
+        self._modify_table_property(name="caching", val=str(prop_val))
```
Comment (roydahan) [inline sdcm/nemesis.py]: Can you please add in comments, what is the default value it starts with and maybe part of documentation that explain how it should affect schema behaviour?

File: sdcm/nemesis.py [hunk 2]
```diff
@@ -445,18 +445,79 @@ def disrupt_nodetool_cleanup(self):
...
+                 "DeflateCompressor")
+        algo = random.choice(algos)
+        prop_val = {"sstable_compression": algo}
+        if algo:
+            prop_val["chunk_length_kb"] = random.choice(["4K", "64KB", "1M"])
+            prop_val["crc_check_chance"] = random.random()
+        self._modify_table_property(name="compression", val=str(prop_val))
+
+    def disrupt_modify_table_crc_check_chance(self):
+        self._modify_table_property(name="crc_check_chance", val=random.random())
```
Comment (roydahan) [inline sdcm/nemesis.py]: Doesn't the value have range?

File: sdcm/nemesis.py [hunk 3]
```diff
@@ -445,18 +445,79 @@ def disrupt_nodetool_cleanup(self):
...
+
+    def disrupt_modify_table_dclocal_read_repair_chance(self):
+        self._modify_table_property(name="dclocal_read_repair_chance", val=random.random())
+
+    def disrupt_modify_table_default_time_to_live(self):
+        self._modify_table_property(name="default_time_to_live", val=630720000)  # max allowed TTL - 20 years
```
Comment (roydahan) [inline sdcm/nemesis.py]: Currently the default isn't set, right?
We should test what happens when rows expires due to TTL and c-s try to read it.

File: sdcm/nemesis.py [hunk 4]
```diff
@@ -445,18 +445,79 @@ def disrupt_nodetool_cleanup(self):
...
+
+    def disrupt_modify_table_read_repair_chance(self):
+        self._modify_table_property(name="read_repair_chance", val=random.random())
+
+    def disrupt_modify_table_speculative_retry(self):
+        options = ("'ALWAYS'",
+                   "'%spercentile'" % random.randint(1, 99),
+                   "'%sms'" % random.randint(1, 1000))
+        self._modify_table_property(name="speculative_retry", val=random.choice(options))
+
```
Comment (roydahan) [inline sdcm/nemesis.py]: @bentsi, despite what we said in the beginning... I think we need the wrap all these new nemesis as ONE nemesis that will randomize across all of them.
We don't really know how all these schema changes will affect our testing and too many schema changes one after another can make the issues impossible to investigate.

Comment (roydahan) [inline sdcm/nemesis.py]: set_current_disruption is missing for all of the above.

File: sdcm/nemesis.py (hunk 2 above)
Comment (bentsi) [reply sdcm/nemesis.py]: it is a probability :) it can be any float in semi-open range [0.0, 1.0)

File: sdcm/nemesis.py (hunk 4 above)
Comment (bentsi) [reply sdcm/nemesis.py]: OK, I will wrap this in on one disrupt method

Comment (bentsi) [reply sdcm/nemesis.py]: actually it is not, see line 454:
```
def _modify_table_property(self, name, val, keyspace="keyspace1", table="standard1"):
    self._set_current_disruption('ModifyTableProperties%s %s' % (name.capitalize(), self.target_node))
```

File: sdcm/nemesis.py (hunk 1 above)
Comment (bentsi) [reply sdcm/nemesis.py]: Will do :+1:

File: sdcm/nemesis.py (hunk 3 above)
Comment (bentsi) [reply sdcm/nemesis.py]: Current default is:
``` default_time_to_live = 0```
I will change it to some random choice in [60, 600]

**Comment by bentsi** [comment]
@roydahan done

File: sdcm/tester.py [hunk 5]
```diff
@@ -21,6 +21,7 @@
 import subprocess
 import datetime
 import platform
+from textwrap import dedent
```
Comment (roydahan) [inline sdcm/tester.py]: We need to add this package to our requirements.

**Comment by roydahan** [comment]
@bentsi, we need to run a 4 days longevity on GCE with this changes and make sure it pass.
We can do it on master if it's stable.
@harelassayag, please work with @bentsi to monitor the cluster.

File: sdcm/tester.py (hunk 5 above)
Comment (bentsi) [reply sdcm/tester.py]: it is in standard Python module library :)

**Comment by bentsi** [comment]
@roydahan should I merge the changes to master?

**Pull request merged**